repose-aggregator/components/filters/ip-user/src/main/scala/org/openrepose/filters/ipuser/IpUserFilter.scala

```diff
@@ -72,7 +71,7 @@ with UpdateListener[IpUserConfig] {
       servletResponse.asInstanceOf[HttpServletResponse].sendError(500)
     } else {
       logger.trace("IP Classification filter handling request...")
-      val request = MutableHttpServletRequest.wrap(servletRequest.asInstanceOf[HttpServletRequest])
+      val request = new HttpServletRequestWrapper(servletRequest.asInstanceOf[HttpServletRequest])
 
       getClassificationLabel(servletRequest.getRemoteAddr).foreach { label =>
         request.addHeader(groupHeaderName, s"$label;q=$groupHeaderQuality")
```

Comment:
I'd rather us use the addHeader method where you pass along the quality instead of using string interpolation to do it in this class.